server/routes/assets/index.js

```diff
@@ -1,7 +1,11 @@
 const serveByClaim = require('../../controllers/assets/serveByClaim');
 const serveByIdentifierAndClaim = require('../../controllers/assets/serveByIdentifierAndClaim');
 
-module.exports = (app) => {
-  app.get('/:identifier/:claim', serveByIdentifierAndClaim);
-  app.get('/:claim', serveByClaim);
+// TODO: Adjust build & sources to use import/export everywhere
+const Actions = require('@actions').default;
+const Sagas = require('@sagas').default;
+
+module.exports = {
+  '/:identifier/:claim': { controller: serveByIdentifierAndClaim, action: Actions.onHandleShowPageUri, saga: Sagas.handleShowPageUri },
```

Comment:
@daovist Take a look at the action/saga binding I've added.
You can also use `httpContext.set('key', value)` as well as `httpContext.get('key')` in controllers.  I'm using this to communicate route information through the stack programmatically.

Reply:
Is this for SSR?

Reply:
Some of it is